module.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/google/tink/go/aead"
 	"github.com/google/tink/go/insecurecleartextkeyset"
 	"github.com/google/tink/go/keyset"
+	certmagicgcs "github.com/grafana/certmagic-gcs/storage"
```

Comment:
better to leave the package name as `storage`

```suggestion
	"github.com/grafana/certmagic-gcs/storage"
```

(and modify below)

Reply:
@hairyhenderson good catch -- unsure why I added the alias 🤔 fixed!